core/src/test/java/com/linecorp/armeria/common/stream/AbortCauseArgumentProvider.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2019 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.common.stream;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Stream;
+
+import org.junit.jupiter.api.extension.ExtensionContext;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.ArgumentsProvider;
+
+class AbortCauseArgumentProvider implements ArgumentsProvider {
+
+    static final List<Throwable> ABORT_CAUSES =
+            Arrays.asList(null, new IllegalStateException("abort stream with a specified cause"));
```

Comment:
Could use `ImmutableList.of()` because `Arrays.asList()` returns a mutable list.

Reply:
At first, I wanted to use it. However `ImmutableList.of()` check not-null for every element. Let me wrap this with `Collections.unmodifiableList()`.